wavelength-plugins/misc.js

```diff
@@ -422,18 +422,32 @@ exports.commands = {
 			if (!Rooms(roomid)) return this.errorReply(`${roomid} is not a room.`);
 			if (Db.disabledScrolls.has(roomid)) return this.errorReply(`${Rooms(roomid).title} has already roomintro scroll disabled.`);
 			msg += '/html <center>' + WL.nameColor(user.name, true) + ' has redeemed roomintro scroll disabler token.<br/>';
-			msg += '<button class="button" name="send" value="/disableintroscroll ' + target[1] + '">Disable Intro Scrool for <b>' + Rooms(roomid).title + '</b></button></center>';
+			msg += '<button class="button" name="send" value="/disableintroscroll ' + target[1] + '">Disable Intro Scroll for <b>' + Rooms(roomid).title + '</b></button></center>';
+			delete user.tokens[target[0]];
+			return WL.messageSeniorStaff(msg);
+		case 'background':
+			if (!target[1]) return this.errorReply('/usetoken background, [img]');
+			target[1] = target[1].trim();
+			msg += '/html <center>' + WL.nameColor(user.name, true) + ' has redeemed a background token.<br/><a href="' + target[1] + '"/><br/>';
```

Comment:
show the image as a small preview, and make clicking the image take you to the full size image (make it a link). THis prevents malicious linking and gives an accurate preview.